hack/build.sh

```diff
@@ -33,7 +33,6 @@ export CGO_ENABLED=0
 export GOARCH="${ARCH}"
 export GOOS="${OS}"
 export GO111MODULE=on
-export GOFLAGS="-mod=vendor"
```

Comment:
why is it removed?

Reply:
Script didn't find authz changes in vendor directory and failed. This was step gap solution to unblock testing. Ideally this should not be required but need to see how can I fix it. Will take it later.